Source/ACE/Entity/Monster.cs

```diff
@@ -23,6 +25,74 @@ public Monster(AceObject aceO)
             else
                 this.WeenieClassid = (ushort)(aceO.WeenieClassId - 0x8000);
 
```

Comment:
Why are we manually copying all this data in a derived type?  Seems like this is a symptom of a bad constructor chain.